package.json

```diff
@@ -6,14 +6,15 @@
     "test": "npm run test-es5 && npm run test-es2015",
     "test-es5": "ng test angular-cld --tsConfig=projects/angular-cld/tsconfig-es5.spec.json",
     "test-es2015": "ng test angular-cld --tsConfig=projects/angular-cld/tsconfig-es2015.spec.json",
-    "test-headless": "npm run test-es5-headless && npm run test-es2015-headless",
+    "test-headless": "npm run build && npm run test-es5-headless && npm run test-es2015-headless",
     "test-es5-headless": "ng test angular-cld --tsConfig=projects/angular-cld/tsconfig-es5.spec.json --browsers ChromeHeadless",
     "test-es2015-headless": "ng test angular-cld --tsConfig=projects/angular-cld/tsconfig-es2015.spec.json --browsers ChromeHeadless",
     "install-sample-from-source": "node ./scripts/install-sample-from-source.js samples/photo_album",
     "install-sample-from-source:jquery": "node ./scripts/install-sample-from-source.js samples/photo_album_with_jquery",
-    "build": "ng build angular-cld && npm run copy-lib-assets",
+    "build": "npm run prepare-version && ng build angular-cld && npm run copy-lib-assets",
     "copy-lib-assets": "cp \"LICENSE\" \"CHANGELOG.md\" \"README.md\" \"dist/angular-cld\"",
-    "bundlewatch": "bundlewatch --config bundlewatch.config.js"
+    "bundlewatch": "bundlewatch --config bundlewatch.config.js",
+    "prepare-version": "node -e \"console.log('export let APP_VERSION = \\'' + process.env.npm_package_version +'\\';')\" > projects/angular-cld/src/lib/version.ts"
```

Comment:
How is this related to the release automation? This is done so we can import the version and use it instead of importing package.json since its not available in angular

Reply:
You need to have a static file with the current version in a specific place. 
I suggest committing this file and update it when when you package for release (this is the only time when a version number is updated)

Reply:
It is a static file with the current version in it. The file is called ./version.ts and its updated when we build the project.. Its ignored when committing as I don't believe there is an added value in having it part of the workspace.

Reply:
Didn't get the last sentence... 
In any case, why do you need to update it every time you run `build`?